src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelAPITests.cs

```diff
@@ -4041,9 +4041,15 @@ static void Main(string[] args)
             var comp = CreateCompilation(source, options: TestOptions.ReleaseExe);
             var tree = comp.SyntaxTrees.Single();
             var model = comp.GetSemanticModel(tree);
+
+            var expected = new bool[2];
```

Comment:
```cs
var expected = new[] { true, false };
```

Reply:
Ooh, was wondering what the inline instantiation syntax was. Thanks!